package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "webxr",
-  "description": "WebXR Device API Specification",
+  "name": "webxr-gamepads",
+  "description": "WebXR Gamepads - Level 1",
```

Comment:
Title nit: Yada yada "Module" yada yada...

Reply:
For the "name" field I was actually sort of intentionally following the CSS WG's pattern of not putting the word "module" there.  Not supper attached either way though.

Reply:
Oh, didn't notice that. I'm all for following whatever conventions the CSS WG has already established.